### ISSUE TITLE
Not raising errors for navigator.hardwareConcurrency or new SharedWorker()

### ISSUE THREAD

**Issue by replete** [amilajack/eslint-plugin-compat#81]
These features are unsupported, listed on caniuse.com, but don't raise errors as expected:

```js
navigator.hardwareConcurrency; // no eslint error
navigator.serviceWorker; // [eslint] navigator.serviceWorker() is not supported in Safari 8, iOS Safari 8, IE 11, Edge 13 (compat/compat)
new SharedWorker(); // no eslint error
```

```
# browserlist

# Desktop:
last 2 Chrome version
last 2 Firefox version
last 2 Edge version
Safari >= 8
IE 11

# Mobile:
last 2 ChromeAndroid version
last 2 FirefoxAndroid version
iOS >= 8
```

```
// package.json
 "eslint-plugin-compat": "1.0.2",
```

```
// .eslintrc.js
module.exports = {
  "env": {
    // because *.spec.js files are in scope:
    "jasmine": true,
    "node": false,
    "browser": true,
    "es6": false
  },
  "globals": {
    // Includes browser globals like FileReader, Blob, etc..:
    "browser": true,
    "$": true,
    "jQuery": true,
    "angular": true,
    "moment": true,
    "_": true,
    "L": true,
    "Chartist": true,
    "saveAs": true
  },
  "plugins": [
    // Looks up 'browserslist' file to see if your code is supported:
    "compat"
  ],
  "rules": {
    "no-bitwise": 2,
    "camelcase": 0,
    // TODO: Cleanup these errors and enforce:
    "curly": 0,
    "eqeqeq": 2,
    "wrap-iife": [ 2, "any" ],
    // TODO: Enforce proper code formatting:
    "indent": [ 0, 2, { "SwitchCase": 1 }],
    "no-use-before-define": [ 2, { "functions": false }],
    // TODO: Investigate whether this should be enforced:
    "new-cap": 0,
    "no-caller": 2,
    "quotes": [ 2, "single" ],
    "no-undef": 2,
    "no-unused-vars": 2,
    // TODO: 'use-strict' should be refactored and enforced:
    "strict": [ 0, "function"],
    // so we can assign 'this' in our controllers :
    "no-invalid-this": 0,
    // Looks up 'browserslist' file to see if your code is supported:
    "compat/compat": 2
  }
}
```

**Comment by djskinner** [comment]
Seeing similar issues with `new MediaRecorder()` and `navigator.getUserMedia`. As with your case `navigator.serviceWorker` correctly returns an error but the rest are ignored.
